app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/SurveyModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.kks.nimblesurveyjetpackcompose.model
+
+data class SurveyModel(
```

Comment:
How about we rename this to `Survey` and rename `entities/Survey` to `SurveyEntity`? 🤔